src/main/java/net/sigmarik/abilitymod/command/TraitCommand.java

```diff
@@ -22,11 +26,12 @@ public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
                 literal("trait").requires(source -> source.hasPermissionLevel(2))
                 .then(literal("add")
                         .then(argument("players", EntityArgumentType.players())
-                        .then(argument("trait", StringArgumentType.word()).suggests(SuggestionProviders.ALL_RECIPES)
+                        .then(argument("trait", StringArgumentType.greedyString())
+                                .suggests(SuggestionProviders.ALL_RECIPES)
```

Comment:
These arguments are not recipes.

Reply:
Should I remind who made this code line?

Reply:
If you noticed an inconvenience and decided to blindly copy it, that's on you, even though I was the one who forgot to remove the test line from the code.